.github/workflows/auto_cherry_pick.yml

```diff
@@ -43,3 +43,37 @@ jobs:
           labels: |
             Auto_Cherry_Picked
             ${{ matrix.to_branch }}
+      - name: Set issue title
+        if: ${{ failure() }}
+        run: |
+            echo "title=[Failed-AutoCherryPick] - ${{ github.event.pull_request.title }}" >> "$GITHUB_ENV"
```

Comment:
You dont need this! You can directly use that tile in `dacbd/create-issue-action@main` GHA for title parameter.

Reply:
I use that title in multiple locations hence I have added that part of env